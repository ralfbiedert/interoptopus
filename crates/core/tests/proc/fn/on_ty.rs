use interoptopus::ffi;

#[ffi]
struct Foo;

#[allow(dead_code)]
fn main() {}
