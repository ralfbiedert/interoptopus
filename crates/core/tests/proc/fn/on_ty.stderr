error: Empty structs are not allowed unless marked as #[ffi(opaque)] or #[ffi(service)].
 --> tests/proc/fn/on_ty.rs:4:8
  |
4 | struct Foo;
  |        ^^^
